index.js

```diff
@@ -64,4 +64,22 @@ validate.addEventListener('submit', (event) => {
     errorMessage.innerHTML = 'Sorry! your mail must be in lowerCase';
     event.preventDefault();
   }
-});
\ No newline at end of file
+});
+// Local storage
+const nameSection = document.querySelector('#name');
```

Comment:
- [ ] Kindly remove the commented code from your codebase as this makes it cleaner